snippets/visualbasic/VS_Snippets_CLR_System/system.Globalization.DateTimeFormatInfo.LongDatePattern/VB/longdatepattern1.vb

```diff
@@ -2,9 +2,10 @@
 Option Strict On
 
 ' <Snippet2>
+Imports System
```

Comment:
It's not necessary to import the System namespace; this Imports statement can be removed.